File: CarlaErik/UvA.SoftCon.Questionnaire/QLS/AST/Model/StyleAttributes/Width.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UvA.SoftCon.Questionnaire.Common.AST.Model;

namespace UvA.SoftCon.Questionnaire.QLS.AST.Model.StyleAttributes
{
    public class Width : StyleAttribute
    {
        public int Value
        {
            get;
            private set;
        }

        internal Width(int value, TextPosition position)
            : base(position)
        {
            Value = value;
        }

        public override T Accept<T>(IQLSVisitor<T> visitor)
        {
            return visitor.VisitWidth(this);
        }
    }
}

File: encoders/qls/src/main/java/nl/uva/sc/encoders/qls/ast/property/Width.java
package nl.uva.sc.encoders.qls.ast.property;

import nl.uva.sc.encoders.ql.ast.TextLocation;

public class Width extends DefaultStyleProperty {

	private final int value;

	public Width(TextLocation textLocation, int value) {
		super(textLocation);
		this.value = value;
	}

	public int getValue() {
		return value;
	}

}
